examples/e-commerce/src/app/app.component.ts

```diff
@@ -10,19 +10,4 @@ export class AppComponent {
     searchClient: algoliasearch('latency', '6be0576ff61c053d5f9a3225e2a90f76'),
     indexName: 'instant_search',
```

Comment:
The config should also include the default `routing`.

Reply:
a77bda8